lib/mayaUsd/ufe/ProxyShapeHierarchyHandler.cpp

```diff
@@ -57,5 +57,14 @@ Ufe::SceneItem::Ptr ProxyShapeHierarchyHandler::createItem(const Ufe::Path& path
 	return fMayaHierarchyHandler->createItem(path);
 }
 
+#ifdef UFE_V2_FEATURES_AVAILABLE
+#if UFE_PREVIEW_VERSION_NUM >= 2022
+Ufe::Hierarchy::ChildFilter ProxyShapeHierarchyHandler::childFilter() const
+{
+	return Ufe::Hierarchy::ChildFilter();
```

Comment:
Return empty list - we don't (yet) support any child filters for the proxy shape.

Reply:
Doesn't this mean that inactive children of the USD pseudo-root (which the proxy shape represents) can't be shown?

Reply:
Oops I forgot about that case. I'll fix it.

Reply:
See commit e19fa54bb6955fb94e66aef00a39385e250dd73a